src/worker/core/GameSim.basketball/PlayByPlayLogger.ts

```diff
@@ -393,7 +200,8 @@ const scoringTypes = [
 	"fgMidRange",
 	"fgMidRangeAndOne",
 	"ft",
-	"tp",
+	"fgTp",
+	"fgTpAndOne",
```

Comment:
Switch back to tp. Ideally use TypeScript to guarantee there are no typos like this. (Also thanks for finding the missing tpAndOne - I wonder if that actually caused any problems...)

Even better - turn the array into a set, cause Set.has is slightly faster than Array.includes.